Makefile

```diff
@@ -19,23 +19,35 @@ lint-fix:
 
 .PHONY: test
 test:
-	pytest evalml/ --doctest-modules --doctest-continue-on-failure
+	pytest evalml/ --doctest-modules --doctest-continue-on-failure --ignore=evalml/tests/automl_tests/dask_tests
```

Comment:
I'd rather we still had one command to run all tests but this is not blocking merge.

Reply:
I'll do a quick PR to reintroduce this.  Good point